src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -508,7 +548,24 @@ public String toString() {
     /* We require jmx access to all nodes in the local datacenter */
     LOCAL,
     /* Each datacenter requires at minimum one reaper instance that has jmx access to all nodes in that datacenter */
-    EACH
+    EACH,
+    /* Sets Reaper in sidecar mode where each Cassandra node has a collocated Reaper instance */
+    SIDECAR;
+
+
+    /**
+     * Check if the current datacenter availability mode is to have collocation between Reaper and a DC/node.
+     * @return true if we're in a collocated mode, false otherwise
+     */
+    public boolean isInCollocatedMode() {
```

Comment:
curious, if there's a more intuitive name for this… like `isOnlyLocalDcJmx()` or  inverted as `allowCrossDcJmx()` , wdyt?

Reply:
It wouldn't really match the sidecar mode IMHO. It's not just about being local to a DC but also a node.

Reply:
that i don't really grok. `EACH` is not restricted to only be local to a node, and `LOCAL` can also be local to a node.

Reply:
I can't think of a better name for now as `LOCAL` is also a form of collocation (to a single DC).